lms/djangoapps/instructor_task/tests/test_tasks.py

```diff
@@ -335,56 +335,6 @@ def test_rescoring_success(self):
             action_name='rescored'
         )
 
-    def test_rescoring_bad_result(self):
-        """
-        Tests and confirm that rescoring does not succeed if "success" key is not an expected value.
-        """
-        input_state = json.dumps({'done': True})
-        num_students = 10
-        self._create_students_with_state(num_students, input_state)
-        task_entry = self._create_input_entry()
-        mock_instance = Mock()
-        mock_instance.rescore = Mock(return_value={'success': 'bogus'})
-        del mock_instance.rescore_problem  # make sure we aren't hitting old CAPA code
-        with patch('lms.djangoapps.instructor_task.tasks_helper.module_state.get_module_for_descriptor_internal') as mock_get_module:
-            mock_get_module.return_value = mock_instance
-            self._run_task_with_mock_celery(rescore_problem, task_entry.id, task_entry.task_id)
-
-        self.assert_task_output(
-            output=self.get_task_output(task_entry.id),
-            total=num_students,
-            attempted=num_students,
-            succeeded=0,
-            skipped=0,
-            failed=num_students,
-            action_name='rescored'
-        )
```

Comment:
They no longer apply, as the return value of `rescore` is no longer used in determining the success of a rescore attempt. Instead we rely on bubbling up exceptions, and that is tested elsewhere.